guides/beginners-course.rst

```diff
@@ -1,15 +1,255 @@
+
+
+Mentoři
+.......
+
+Hlavní kouč
+
+    Na každé lekci by měl být jeden „hlavní kouč”, který lekci řídí
+    a zná příslušné materiály.
+    Je dobré, když tuhle funkci po celé trvání kurzu zastává ten stejný
+    člověk, který má pak přehled o tom, co se probralo.
+    Pokud se hlavní koučové střídají, měl by každou lekci vést někdo, kdo
+    byl minimálně na lekci předchozí.
+
+Vedlejší koučové
+
+    Ostatní mentoři pomáhají (např. rozdávají papírové materiály),
+    a hlavně řeší konkrétní problémy.
+    V téhle roli nemusí být všichni experti. Naopak, pro účastníka
+    předchozího „běhu” začátečnického kurzu je koučování skvělá příležitost
+    jak si všechno zopakovat, a zjistit kolik se toho naučil :)
+    
+    Doporučujeme zhruba jednoho vedlejšího kouče na 3-5 účastníků.
+    Ze začátku, na instalaci a nastavení prostředí, je lepší jich pozvat spíš
+    víc; potom nebývá problém když občas někdo odpadne.
+
+    Koučové se dají nabrat na srazech, případně se dají oslovit programátoři
+    v okolních firmách.
+    Určitě koučování nabídni i účastníkům předchozích kurzů.
+
+Obsah
+.....
+
+Naše materiály jsou `volně k použití <http://pyladies.cz/course.html>`_
+pod licencí CC-BY-SA, a dají se použít přímo nebo upravit přesně podle
+vašich představ :)
+
+
+Rozvrh
+------
+
+Vyber si den D, kdy má kurz začít.
+Podle toho si vytvoř časový rozvrh, ať na nic nezapomeneš:
+
+* D - 2 měsíce
+
+    * Začít shánět kouče (pokud je už nemáš potvrzené)
+
+* D - 1 měsíc
+
+    * Zveřejnit přihlašovací formulář.
+
+      (XXX: Odkaz na vzorovou přihlášku)
+
+* D - 2 týdny
+
+    * Vybrat účastníky
+
+        * Vybraným poslat mail s radostnou zprávou
+        * Zhruba pět až deset odmítnutých informovat, že jsou náhradníci
+        * Ostatní odmítnutým sdělit že se bohužel nevejdou
+    * Informovat kouče
+
+    (XXX: šablony na tyhle maily)
+
+* D - 1 den
+
+    * Poslat účastníkům připomínací mail
+
+* D + zhruba 3 měsíce
+
+    * Plánovaný konec kurzu
+
+
+První lekce
+-----------
+
+První lekce je u nás jiná než ostatní: je potřeba všechno nainstalovat
+a nastavit (Python, Git, editor, virtualenv).
+To se na každém počítači dělá jinak, a každému to trvá jinak dlouho.
+(Dá se říct že cíl první lekce je, aby po ní fungovaly všechny počítače stejně.)
+
+Proto doporučujeme první lekci kromě úvodní prezentace neřídit centrálně,
+ale nechat účastníky, aby (s pomocí koučů) všechno nainstalovali podle
+instrukcí na internetu.
+
+Součást první lekce je i seznámení s příkazovou řádkou (která je pak potřeba
+k nastavení virtuálního prostředí).
+
+V rámci úvodní prezentace by mělo padnout:
+
+    * Trocha historie
+    * Co se naučíme, a co naopak v kurzu nebude
+    * Instrukce pro účastníky – viz barevné papírky níže
+    * Pár instrukcí pro kouče
+    * Představení koučů
+
+Osvědčil se systém barevných papírků, viz dole.
+
+
+Systém barevných papírků
+------------------------
+
+Osvědčil se nám „papírkový” systém převzatý ze Software Carpentry.
```

Comment:
No jo. Problém je že oni o papírkách nikde moc nepíšou; říkal to jeden z nich na nějaké konferenci ale už si nepamatuju co to bylo za prezentaci :(